src/components/secure-backup/restore-backup/index.tsx

```diff
@@ -44,7 +44,12 @@ export class RestoreBackup extends React.Component<Properties, State> {
           </p>
 
           <div {...cn('input-container')}>
-            <Input placeholder='Enter your recovery key' onChange={this.trackRecoveryKey} value={this.recoveryKey} />
+            <Input
+              {...cn('input', `${this.props.errorMessage && 'error'}`)}
```

Comment:
Is there not already a way to tell the input that it's in an error state?

Reply:
At some point during redesign, the border was removed from `Input` in `zUI`. There is an error state for the component that sets the border color, but I'll need to go into `zUI` and add `border: 1px solid` to that error state. This was more of a quick fix for now.